test/helper.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Knex = require('../lib/libraries/knex');
+
+const firstMovie  = { title: 'The Prestige', release_year: 2006 };
+const secondMovie = { title: 'Memento' };
+const thirdMovie  = { title: 'The Departed', release_year: 2006 };
+
+exports.firstMovie  = firstMovie;
+exports.secondMovie = secondMovie;
+exports.thirdMovie  = thirdMovie;
+
+beforeEach(() => {
+  return Knex('movies').truncate();
+});
+
+beforeEach(() => {
+  return Knex('movies').insert([firstMovie, secondMovie, thirdMovie]);
```

Comment:
We usually only do truncates at this high level. The inserts are done in each test file because different tests may rely on different movies being inserted and it's easier to follow the state of the database if the inserts happen in the test file. Let me know if it's easier to talk about the ideal structure in person.